packages/core/src/MachineNode.ts

```diff
@@ -142,22 +142,21 @@ export class MachineNode<
   }
 
   /**
-   * Clones this state machine with custom options and context.
```

Comment:
any reason behind the removal of this? you still support passing `context` in this method

Reply:
Reverted